assignments/assignment_4/README.md

```diff
@@ -1,25 +1,15 @@
-# Assignment 4: Design Patterns
+# Assignment 4: OOP and VSCode Shortcuts
 
-The focus of this (final) assignment is about OOP and design patterns. And we have bad news: we went to check the Eurostat source and now they are using JSON as a data format. Also, the file was zipped.
+The focus of this assignment is about OOP. In particular, we want to make sure that we only provide valid values for our countries. This is the perfect scenario to make use of the Enum class. Note that this is a special case of classes since they are rarely instantiated. 
```

Comment:
```suggestion
The focus of this assignment is about OOP. In particular, we want to make sure that we only provide valid values for our countries. This is the perfect scenario to make use of the Enum class. 
```

**Suggestion**: Remove this so we don't confuse students. Enum classes can be instantiated.. and it's how we can use enums to validate things: For example using `Region("PT")` will yield the class variable `Region.PT` which is treated as an instance, but is it? It can be confusing...